spec/features/root/root_spec.rb

```diff
@@ -7,17 +7,18 @@
     user_3 = User.create!(name: 'user_3', email: 'email3@gmail.com')
     visit root_path
 
-    within '.title' do 
+    within '#title' do 
       expect(page).to have_content('Welcome to Viewing Party')
     end 
 
-    within '.new-user' do 
+    within '#new-user' do 
       expect(page).to have_button 'Register' 
       click_button 'Register'
       expect(current_path).to eq(new_user_path)
```

Comment:
These are really cool ways to handle routing that I hadn't seen before, i'd love to use them myself. If you see a PR from me that could be refactored to use them don't hesitate to let me know!!